keyboards/nightly_boards/octopadplus/info.json

```
@@ -0,0 +1,39 @@
+{
+    "keyboard_name": "Octopad+"	,
+    "manufacturer": "DeskDaily",
+    "url": "",
+    "maintainer": "DeskDaily",
+    "usb": {
+    	"vid": "0xD812",
+    	"pid": "0x0014",
+    	"device_version": "0.0.1"
+	},
+	    "rgblight": {
+        "animations": {
+            "all": true
```

Could you explicitly define which animations are enabled? 
```suggestion
            "alternating": true,
            "breathing": true,
            "christmas": true,
            "knight": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "rgb_test": true,
            "snake": true,
            "static_gradient": true,
            "twinkle": true
```

---

Will do thanks